lib/exabgp/rib/store.py

```diff
@@ -54,7 +54,7 @@ def clear (self):
 
 	def sent_changes (self, families=None):
 		# families can be None or []
-		requested_families = self.families if not families else set(families).intersection(self.families)
+		requested_families = self.families if families == None else set(families).intersection(self.families)
```

Comment:
should be "families is None" if you do not mind changing it .. Otherwise Happy to merge.